lib/msfdb_helpers/db_interface.rb

```diff
@@ -13,10 +13,6 @@ def delete
       raise NotImplementedError
     end
 
-    def reinit
-      raise NotImplementedError
-    end
-
```

Comment:
Removed `reinit` as it seemed like logic that needed to exist in msfdb, rather than in these helpers